### PR TITLE
Ws2 1288 update intro content market labels

### PR DESCRIPTION
**Pull request by mlsamuelson**
Updates to include the UDS-1158 fixes for WS2-1288 and tweaks labels and help text to be more informative about behaviors to expect in overriding degree market text.

### CODE REVIEW

File: config/install/field.field.node.degree_detail_page.field_degree_dtl_intro_content.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ field_name: field_degree_dtl_intro_content
 entity_type: node
 bundle: degree_detail_page
 label: 'Intro content'
-description: ''
+description: 'Overrides default market text.'
```
Comment (tbutterf) [inline config/install/field.field.node.degree_detail_page.field_degree_dtl_intro_content.yml:24]: Maybe say "Overrides default marketing text retrieved from Degree Search," or something?

File: config/install/field.field.node.degree_detail_page.field_deg_dtl_intro_hide_market.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@ id: node.degree_detail_page.field_deg_dtl_intro_hide_market
 field_name: field_deg_dtl_intro_hide_market
 entity_type: node
 bundle: degree_detail_page
-label: 'Hide market text'
+label: 'Hide market text (includes custom Intro content)'
```
Comment (tbutterf) [inline config/install/field.field.node.degree_detail_page.field_deg_dtl_intro_hide_market.yml:14]: I would replace "market" with "marketing," and "includes" with "including."

**Review by tbutterf — APPROVED** [review]
I have left a couple of comments with wordsmithing options, but I think it's fine as-is. Feel free to ignore the suggestions, or implement them. :-)

File: config/install/field.field.node.degree_detail_page.field_deg_dtl_intro_hide_market.yml (hunk 2 above)
Comment (mlsamuelson) [reply config/install/field.field.node.degree_detail_page.field_deg_dtl_intro_hide_market.yml:14]: Leaving as "includes" works better to capture nuance of field handling.

**Pull request merged**